app/app.go

```diff
@@ -1262,6 +1271,13 @@ func VerifyAddressFormat(bz []byte) error {
 	return nil
 }
 
+func (app *App) ProcessProposalVerifyTx(txBz []byte) (sdk.Tx, error) {
```

Comment:
this don't work, because the `NoOpProcessProposal` don't call it.

Can we discuss before overriding the code directly, what's the issue with the version before?

Reply:
Yes, no issue if we stick to NoOpMempool for now.

Reply:
this method is not called by anyone, process proposal logic is still none, I feel you has some misunderstanding here?